pallets/parachain-staking/src/weights.rs

```diff
@@ -124,77 +122,65 @@ pub trait WeightInfo {
 /// Weights for parachain_staking using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	// Storage: ParachainStaking DelegatorState (r:1 w:1)
-	#[rustfmt::skip]
-	fn hotfix_remove_delegation_requests(x: u32, ) -> Weight {
-		(0 as Weight)
-			// Standard Error: 5_000
-			.saturating_add((7_270_000 as Weight).saturating_mul(x as Weight))
-			.saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(x as Weight)))
-			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(x as Weight)))
-	}
 	// Storage: ParachainStaking CandidateInfo (r:4 w:0)
 	// Storage: ParachainStaking CandidatePool (r:1 w:1)
 	#[rustfmt::skip]
 	fn hotfix_update_candidate_pool_value(x: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 174_000
-			.saturating_add((24_976_000 as Weight).saturating_mul(x as Weight))
+			// Standard Error: 72_000
+			.saturating_add((14_273_000 as Weight).saturating_mul(x as Weight))
```

Comment:
This is about a 50% reduction but it's not obvious why. Is this expected, or is it just noise?

Reply:
Yeah this is strange, seems like noise to me. This extrinsic wasn't even affected by the change. The number of reads and writes remain the same.

```rust
                /// Hotfix patch to correct and update CandidatePool value for candidates that have
		/// called candidate_bond_more when it did not update the CandidatePool value
		pub fn hotfix_update_candidate_pool_value(
			origin: OriginFor<T>,
			candidates: Vec<T::AccountId>,
		) -> DispatchResultWithPostInfo {
			frame_system::ensure_root(origin)?;
			for candidate in candidates {
				if let Some(state) = <CandidateInfo<T>>::get(&candidate) {
					Self::update_active(candidate, state.total_counted);
				} // else candidate is not a candidate so no update needed
			}
			Ok(().into())
		}
```

If we're having ~50% variance in our benchmarked weights, it's certainly a cause for concern. I'll run them again and see how it comes up this time.

Reply:
I re-ran the benchmark
https://github.com/PureStake/moonbeam/compare/nish-staking-no-snipe...nish-staking-no-snipe-weights